.gitignore

```diff
@@ -107,5 +107,6 @@ datasets/qm8.csv
 env.yml
 .python-version
 **/*.ipynb_checkpoints/
+.virtual_documents
```

Comment:
Please do some research on what this .virtual_documents

Reply:
It temporarily stores jupyter notebooks when using an lsp server with jupyter labs.

our backend Jupyter server extension creates a temporary file on the file system (by default in the .virtual_documents directory). [1]

References
----
[1].. https://jupyter.org/enhancement-proposals/72-language-server-protocol/language-server-protocol.html